lib/index.d.ts

```diff
@@ -68,7 +68,7 @@ declare namespace Component {
     $app: AppState;
 
     /** Attributes parsed from component's html attributes using attrsSchema */
-    $attrs: {[attr: string]: any};
+    $attr: {[attr: string]: any};
```

Comment:
This was my oversight. 

$attr is a function so it should be 

```suggestion
   $attr<A extends keyof Attrs>(attr: A): Attrs[A]; 
```

You’ll need to pass Attr from component generic to here and give ConfigOptions an optional Attrs = AnyAttrs as well.